pascals_triangle.py

```diff
@@ -0,0 +1,41 @@
+Time and space complexity = O(NumRows^2)
```

Comment:
Why is space O(n^2)?

Reply:
If number of rows is 5, we will need a list of lists of size 5*5 to provide the result, unless we are not counting it?